modules/alerts/outputs.tf

```
@@ -0,0 +1,21 @@
+######################################################
+# Outputs
+######################################################
+
+// 'error_level_emitted_when' indicates the event which will trigger the alert if the default 'alert' variable is provided.
+// 'info_level_emitted_when' indicates the events which will trigger the alert
+
+output "alert_topics" {
+  description = "SNS topic ARNs of alerts used for monitoring a Common Fate deployment"
+  value = {
+    deployment_failures = {
+      emitted_when  = "An ECS deployment has failed"
+      sns_topic_arn = aws_sns_topic.deployment_failures.arn
+    }
+
+    job_failures = {
+      errors_emitted_when = "A background job has failed"
```

In this output what was your intention for the key here, I see there are a few variants in comments and implementation

```suggestion
      errors_emitted_when = "A background job has failed"
```

---

thanks I’ll tidy this up. Originally I had implemented this such that users could route both “info” level events (such as a job being completed) as well as failures to each SNS topic, with the idea that you could use the info events to test your alerts. The info events were separate EventBridge rules to the failure ones.

I realized though that this could cause more harm than good, there is a risk that the info events could fire correctly but failure events wouldn’t, and then you’d have a false sense that the alerts were working correctly.